.github/workflows/coding-standards.yml

```diff
@@ -53,3 +56,61 @@ jobs:
       # The -q option is required until phpcs v4 is released
       - name: "Run PHP_CodeSniffer"
         run: "vendor/bin/phpcs -q --no-colors --report=checkstyle | cs2pr"
+
+      - name: "Format the code"
+        run: ./vendor/bin/phpcbf
```

Comment:
Some errors can't be fixed. We need to run `./vendor/bin/phpcs` after this command to avoid committing partially fixed code.